pyclip/win_clip.py

```diff
@@ -149,7 +149,12 @@ def clear(self) -> None:
             clip.EmptyClipboard()
         return
 
+    def _handle_dibv5(self, data):
+        return data.rstrip(b'\x00\x00')
```

Comment:
Though, this won't work if the data literally ends in `\x00\x00` since it will strip too much.